### PR TITLE
Fix some broken references

### PR DESCRIPTION
**Pull request by sfuhrer**

### CODE REVIEW

File: en/config_fw/advanced_tuning_guide_fixedwing.md [hunk 1]
```diff
@@ -79,9 +79,8 @@ Furthermore, these two values define the height rate limits commanded by the use
...
-All L1 parameters are described [here](../advanced_config/parameter_reference.md#fw-l1-control). 
+All L1 parameters are described [here](../advanced_config/parameter_reference.md#fw-path-control). 
 
-- [FW_L1_PERIOD](../advanced_config/parameter_reference.md#FW_L1_PERIOD) - This is the L1 distance and defines the tracking point ahead of the aircraft it's following. 
-  A value of 25 meters works for most aircraft.
-  A value of 16-18 will still work, and provide a sharper response.
-  Shorten slowly during tuning until response is sharp without oscillation.
+- [NPFG_PERIOD](../advanced_config/parameter_reference.md#NPFG_PERIOD) - This is the previously called L1 distance and defines the tracking point ahead of the aircraft it's following. 
+  A value of 10-20 meters works for most aircraft.
+  Shorten slowly during tuning until response is sharp without oscillation. Vehicles with a slow roll dynamic should have this value increased.
```
Comment (hamishwillee) [inline en/config_fw/advanced_tuning_guide_fixedwing.md:86]:
```suggestion
  Shorten slowly during tuning until response is sharp without oscillation.
  Vehicles with a slow roll dynamic should have this value increased.
```

File: en/config_fw/advanced_tuning_guide_fixedwing.md [hunk 2]
```diff
@@ -79,9 +79,8 @@ Furthermore, these two values define the height rate limits commanded by the use
 
 ### L1 Controller Tuning (Position)
```
Comment (hamishwillee) [inline en/config_fw/advanced_tuning_guide_fixedwing.md:80]: Are they still referred to as "L1" ? I.e. should we rename FW Path Control Parameters?

File: en/config_vtol/vtol_quad_configuration.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ For airframe specific documentation and build instructions see [VTOL Framebuilds
 
 1. Run *QGroundControl*
 2. Flash the firmware for your current release or master (PX4 `main` branch build).
-3. In the [Airframe](config/airframe.md) section select the appropriate VTOL airframe.
+3. In the [frame configuration](../airframes/airframe_reference.md) section select the appropriate VTOL airframe.
```
Comment (hamishwillee) [inline en/config_vtol/vtol_quad_configuration.md:11]: I think this was incorrect link to the right place. It is supposed to point 

```suggestion
3. In the [Frame setup](../config/airframe.md) section select the appropriate VTOL airframe.

```

File: en/config_fw/advanced_tuning_guide_fixedwing.md (hunk 2 above)
Comment (sfuhrer) [reply en/config_fw/advanced_tuning_guide_fixedwing.md:80]: Ah sorry missed that, yes, your suggestion is better

File: en/config_fw/advanced_tuning_guide_fixedwing.md [hunk 4]
```diff
@@ -79,9 +79,9 @@ Furthermore, these two values define the height rate limits commanded by the use
...
```
Comment (hamishwillee) [inline en/config_fw/advanced_tuning_guide_fixedwing.md:80]:
```suggestion
### FW Path Control Tuning (Position)
```

File: en/config_fw/advanced_tuning_guide_fixedwing.md [hunk 5]
```diff
@@ -79,9 +79,9 @@ Furthermore, these two values define the height rate limits commanded by the use
...
 
-All L1 parameters are described [here](../advanced_config/parameter_reference.md#fw-l1-control). 
+All L1 parameters are described [here](../advanced_config/parameter_reference.md#fw-path-control).
```
Comment (hamishwillee) [inline en/config_fw/advanced_tuning_guide_fixedwing.md:82]:
```suggestion
All path control parameters are described [here](../advanced_config/parameter_reference.md#fw-path-control). 
```

**Comment by hamishwillee** [comment]
Thanks very much! Merging

**Pull request merged**